dcicutils/bundle_utils.py

```diff
@@ -55,38 +59,41 @@ def __repr__(self):
 
 
 class BoolHint(TypeHint):
+    def __init__(self):
+        super().__init__()
```

Comment:
Is this not the default behavior?

Reply:
No it's not. A derived __init__ must explicitly call its base __init__, if it wants that behavior.